tree/ntuple/v7/test/ntuple_extended.cxx

```diff
@@ -259,3 +259,84 @@ TEST(RNTuple, LargeFile2)
    }
 }
 #endif
+
+namespace {
+
+int gTestLastLevel = -1;
+bool gTestLastAbort = false;
+std::string gTestLastLocation;
+std::string gTestLastMsg;
+
+static void TestErrorHandler(int level, Bool_t abort, const char *location, const char *msg)
+{
+   gTestLastLevel = level;
+   gTestLastAbort = abort;
+   gTestLastLocation = location;
+   gTestLastMsg = msg;
+}
```

Comment:
Let's use `ROOT::TestSupport::CheckDiagsRAII` (in `ROOT/TestSupport.hxx`) instead, which should warn if there are unexpected errors/warnings.